ci-operator/config/oadp-qe/oadp-qe-automation/oadp-qe-oadp-qe-automation-main__oadp1.1-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,77 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  mtc-python-client:
+    name: mtc-python-client
+    namespace: oadp-qe
+    tag: master
+  oadp-apps-deployer:
+    name: oadp-apps-deployer
+    namespace: oadp-qe
+    tag: master
+  oadp-e2e-qe:
+    name: oadp-e2e-qe
+    namespace: oadp-qe
+    tag: release-v1.1
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/testing/interop/Dockerfile
+  inputs:
+    mtc-python-client:
+      paths:
+      - destination_dir: mtc-python-client.tar.gz
+        source_path: /mtc-python-client.tar.gz
+    oadp-apps-deployer:
+      paths:
+      - destination_dir: oadp-apps-deployer.tar.gz
+        source_path: /oadp-apps-deployer.tar.gz
+    oadp-e2e-qe:
+      paths:
+      - destination_dir: oadp-e2e-qe.tar.gz
+        source_path: /oadp-e2e-qe.tar.gz
+  to: oadp-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: oadp-interop-aws
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      OPERATORS: |
+        [
+          {"name": "redhat-oadp-operator", "source": "redhat-operators", "channel": "stable-1.1", "install_namespace": "openshift-adp", "target_namespaces": "openshift-adp", "operator_group":""},
+          {"name": "volsync-product", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-operators", "target_namespaces": "", "operator_group":"global-operators"},
+          {"name": "odf-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-storage", "target_namespaces": "openshift-storage", "operator_group":"openshift-storage"}
+        ]
+    test:
+    - ref: install-operators
+    - ref: oadp-annotate-volsync
+    - ref: oadp-execute-tests
+    workflow: oadp-ipi-aws
```

Comment:
Why create a new workflow rather than just exploding the ipi-aws steps here as pre & post?

Like @chetna14manku as done [here](https://github.com/openshift/release/pull/36871/files#diff-dcc1c6f6c814d60c4e9402d430743cfb190f7a8fdcb1aaab05a3bee3ecb533ffR47)

Or I have done with OPP [here](https://github.com/openshift/release/blob/master/ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml#L50)

Reply:
Right I see what you mean thanks for the explanation